nmtwizard/serving.py

```diff
@@ -282,24 +286,21 @@ def preprocess_example(preprocessor, index, raw_example, config=None):
         raise ValueError("Using both a target prefix and a fuzzy target is currently unsupported")
     if target_prefix is not None:
         target_text = target_prefix
-        target_type = "prefix"
```

Comment:
Can't the example simply contain "target" field and "type" or "name" field ?
Will it break the compatibility with other components ?
This "fuzzy" coming out of the blue feels a bit strange.

Reply:
I don't think we can easily change the server API.

Reply:
In that case, maybe add the "name" directly here and pass it on to the preprocessor, instead of having a special target_type structure ? What do you think ?

Reply:
I'm not sure. The target name is a concept that is specific to the TU structure, and the current serving code has no knowledge about the TU. So I think it's better to not move TU related logics inside this file.